fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/api/SavingsAccountTransactionsApiResourceSwagger.java

```diff
@@ -20,11 +20,172 @@
+                public Integer decimalPlaces;
+                @Schema(example = "0")
+                public Integer isMultiplesOf;
+                @Schema(example = "$")
+                public String displaySymbol;
+                @Schema(example = "currency.USD")
+                public String nameCode;
+                @Schema(example = "US Dollar ($)")
+                public String displayLabel;
+            }
+
+            static final class GetTransactionsPaymentDetailData {
+
+                private GetTransactionsPaymentDetailData() {}
+
+                static final class GetPaymentTypeData {
+
+                    private GetPaymentTypeData() {}
+
+                    @Schema(example = "1")
+                    public Long id;
+                    @Schema(example = "Money Transfer")
+                    public String name;
+                    @Schema(example = "false")
+                    public Boolean isSystemDefined;
+
+                }
+
+                @Schema(example = "1")
+                public Long id;
+                public GetPaymentTypeData paymentType;
+                @Schema(example = "acc123")
+                public String accountNumber;
+                @Schema(example = "che123")
+                public String checkNumber;
+                @Schema(example = "rou123")
+                public String routingCode;
+                @Schema(example = "rec123")
+                public String receiptNumber;
+                @Schema(example = "ban123")
+                public String bankNumber;
+            }
+
+            static final class GetSavingsAccountChargesPaidByData {
+
+                private GetSavingsAccountChargesPaidByData() {}
+
+                @Schema(example = "1")
+                public Long chargeId;
+                @Schema(example = "0")
+                public BigDecimal amount;
+            }
+
+            @Schema(example = "1")
+            public Long id;
+            public GetTranscationEnumData transactionType;
+            @Schema(example = "1")
+            public Long accountId;
+            @Schema(example = "000000001")
+            public String accountNo;
+            @Schema(example = "[2023, 05, 01]")
+            public LocalDate date;
+            public GetTransactionsCurrency currency;
+            public GetTransactionsPaymentDetailData paymentDetailData;
+            @Schema(example = "500")
+            public BigDecimal amount;
+            @Schema(example = "500")
+            public BigDecimal runningBalance;
+            @Schema(example = "false")
+            public boolean reversed;
+            @Schema(example = "[2023, 05, 01]")
+            public LocalDate submittedOnDate;
+            @Schema(example = "false")
+            public boolean interestedPostedAsOn;
+            @Schema(example = "mifos")
+            public String submittedByUsername;
+            @Schema(example = "false")
+            public boolean isManualTransaction;
```

Comment:
use `Boolean` class instead

Reply:
@adamsaghy  Thanks for the feedback. Will fix it.

Reply:
@adamsaghy The respective response entity is having this field as primitive boolean.

Reply:
it is fine then :)